lib/elixir/lib/version.ex

```diff
@@ -51,10 +51,12 @@ defmodule Version do
 
       "~> 2.0.0"
 
-  `~>` will never include pre-release versions of its upper bound.
+  `~>` will never include pre-release versions of its upper bound,
+  regardless of the usage of the `allow_pre:` option, or whether the operand
+  is a pre-release version.
   It can also be used to set an upper bound on only the major
```

Comment:
Turn this into a single paragraph "...is a pre-release version. It can also..."? 

(or add a new line so they are two separate paragraphs?)

Reply:
Yes, it's one paragraph. thank you.